arelle/plugin/validate/EFM/Filing.py

```diff
@@ -1029,7 +1030,12 @@ def sevMessage(sev, messageKey=None, **kwargs):
                 
             sevs = deiValidations["sub-type-element-validations"]
             deiCAxes = deiValidations["axis-validations"]["c"]["axes"]
-            deiDefaultPrefixedNamespaces = deiValidations["prefixed-namespaces"]
+            # Its possible that extension concepts could have prefixes that match `cef` or `vip`
+            # and EFM.6.5.55 or EFM.6.5.56 validations so we exclude all extension namespaces by
+            # filtering out prefix namespace combos where the namespace matches known SEC domains.
+            deiDefaultPrefixedNamespaces = {
+                prefix: qname for prefix, qname in deiValidations["prefixed-namespaces"].items() if secDomainPattern.search(qname)
```

Comment:
These aren't qnames?
```suggestion
                prefix: namespace for prefix, namespace in deiValidations["prefixed-namespaces"].items() if secDomainPattern.search(namespace)
```